flint/local_project_test.go

```diff
@@ -166,6 +166,29 @@ func TestLocalProjectFindsTestScript(t *testing.T) {
 	}
 }
 
+var CodeOfConductTests = []scenarios{
+	{"", false},
+	{"CODE_OF_CONDUCT", true},
+	{"CODE_OF_CONDUCT.md", true},
```

Comment:
@derickr Looks like the code indeed supports that. Feel free to send over a patch to make the tests more invention revealing.